### PR TITLE
Rs 2022 03 13

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
 -- 「……そうですか。そっちの横やりで落としたものですけど」
-"...Is that so? But I believe it was you that made me drop them."
+"...Is that so.
+ I seem to recall dropping them when you barged in."
 }
 [sha:514e4114d4443af814902d477bf49b1deb79b038]{
 -- Page 547, Offset 21147.
 -- ノエルから眼鏡を受け取って、かけ直す。
-I take the glasses from Noel and put them on.
+I take the glasses from Noel, and put them on.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:45]: revert imo

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 2]
```diff
@@ -46,17 +52,20 @@ I take the glasses from Noel and put them on.
...
+Just what am I supposed to make of this situation?
 }
 [sha:4d851e3607f717a1433527aecee6bec85c739124]{
 -- Page 548, Offset 21150.
 -- 「とまあ、美少年がひとりで出歩くと、こんなふうに危険がいっぱいなワケ。忠告がてら脅かしちゃったけど、勢い食べちゃうところだったわ」
-"A handsome young boy going out alone will bring forth many dangers like this. I tried to warn you while being threatening, but I almost ended up eating you up instead."
+"Well,
+ These are the sorts of dangers that lie in wait for handsome young men that go wandering about on their own.
+ I was just trying to give you a little warning to scare you off,
+ but I very nearly ended up having my way with you instead."
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:63]: what a lame character

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 3]
```diff
@@ -126,17 +140,22 @@ Regardless of her explanation, I'm starting to get a handle on the situation.
 [sha:2910cc7549526b9a3e0bf95f405450b33c03256d]{
 -- Page 553, Offset 21165.
 -- 志貴クンは無神論者でしょ？　だから実感は湧かないだろうけど、私たち教会は主の御名のもと、人の善き営みを守る意思がある。たとえそれが、こんな末法の国でもね」
-You're an atheist, right? So it might be hard to believe, but our Church is dedicated to protecting the good of mankind in the name of the Lord. Even in a country under the latter day of the Dharma."
+// 末法 is Latter Day of the Dharma but uh that seems a bit obtuse as a reference to me?
+You're an atheist, right?
+ So this may not have sunk in yet,
+ but those of the Church are dedicated to protecting the works of man in the name of our Lord.
+ Even in a decadent country, like the one we're in now."
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:147]: I don't really approve of this change. [The Latter Day of the Dharma](https://en.wikipedia.org/wiki/Three_Ages_of_Buddhism#Latter_Day_of_the_Dharma) is a pretty important concept in Buddhism afaik. Moreover, I think "decadent" is a little soft. The English definitions I could find that didn't use the word Dharma used the phrase ["degenerate"](https://encyclopediaofbuddhism.org/wiki/Three_Ages_of_Dharma).

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 4]
```diff
@@ -191,12 +216,14 @@ Her logic certainly makes sense.
...
 -- でも前提が間違えている。
-But her underlying hypothesis is wrong.
+But she's basing it on a flawed premise.
 }
 [sha:5d0eacd5ef61d27903062b3e9f62cea8b5533b48]{
 -- Page 559, Offset 21179.
 -- 俺はアルクェイドに従っている訳じゃないし、そもそもあいつは善い吸血鬼だ。彼女と同じように、街を守るために吸血鬼を追っているのに―――
-It's not like I'm following Arcueid's orders. And besides, she's a good vampire. Arcueid is going after vampires in order to protect the city just like you, Noel―――
+It's not like I'm blindly following her orders.
+ And besides, Arcueid is a good vampire.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:225]: drop and

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 5]
```diff
@@ -311,12 +356,17 @@ This time, I confront her with just my gaze without putting myself on guard.
...
 }
 [sha:8487a57d1a78aa3075c4f52a2af4b4d71a120498]{
 -- Page 569, Offset 21203.
 -- 「…………そ。キミが噛まれていたなら、そんな戯言と一緒に手足ブッた斬って置物にしてるところだけど。
-"...I see. If you had been bitten, I would have ripped you limb from limb and turned you into an ornament along with your bullshit.
+// 'object placed in tokonoma' -> object on the mantel
+"...I see.
+ If you had been bitten,
+ I'd have ripped you limb from limb,
+ stuffed you and put you up on the mantel next to the bullshit you just spouted.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:369]: oxford comma

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 6]
```diff
@@ -491,15 +554,17 @@ Before I lose consciousness and before my feet give in, causing my entire body t
...
 -- Page 580, Offset 21239.
 -- 全身が弛緩していくおぞましさ。
-and the repulsiveness of my entire body becoming loose.
+and the disquieting sensation of my body going limp.
 }
 [sha:570cbe9f84f975ea96a93efa46605f91613f2b20]{
 -- Page 580, Offset 21240.
 -- 俺は塀に背中を預けたまま、うっすらと眠りに落ちていく自分自身を見つめていた―――
-With my back still against the wall, I watch myself as I slowly drift into a state of sleep―――
+// Is this supposed to be out-of-body?
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:567]: definitely

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:63]: 2022-03-12-204946_1920x1140_scrot

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:147]: Fair enough, we can keep it. Religious terms are fair game from a Church person I suppose

**Pull request merged**